src/Compositor.cpp

```diff
@@ -1195,6 +1195,17 @@ CWindow* CCompositor::getFirstWindowOnWorkspace(const int& id) {
     return nullptr;
 }
 
+CWindow* CCompositor::getTopLeftWindowOnWorkspace(const int& id) {
+    for (auto& w : m_vWindows) {
+        if (w->m_iWorkspaceID != id || !w->m_bIsMapped || w->isHidden())
+            continue;
+        const auto WINDOWIDEALBB = w->getWindowIdealBoundingBoxIgnoreReserved();
+        if (WINDOWIDEALBB.x == 1 && WINDOWIDEALBB.y == 1)
```

Comment:
spotted this: this will break on multimon. We should check against the workspace's monitor's pos, not 1

Reply:
fixd